hubgrep/frontend_blueprint/forms/hosting_service/hosting_services.py

```diff
@@ -0,0 +1,164 @@
+import json
+import logging
+
+import markdown
+
+from flask_wtf import FlaskForm
+from wtforms import StringField, SelectField, TextAreaField
+from wtforms.widgets.html5 import URLInput
+from wtforms.validators import ValidationError, URL
+from wtforms.validators import InputRequired
+
+from hubgrep.lib.hosting_service_interfaces import hosting_service_interface_mapping
+from hubgrep.models import HostingService
+
+from hubgrep.frontend_blueprint.forms.hosting_service.validators import (
+    validate_custom_config,
+    validate_url,
+    validate_api_url,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+class NoHostingServiceFormException(Exception):
+    pass
```

Comment:
thrown only if we forget to implement a form for a new service, or a user messes with the "type" select box